packages/libs/components/src/stories/plots/NetworkPlot.stories.tsx

```diff
@@ -171,6 +177,120 @@ WithActions.args = {
   getNodeMenuActions: getNodeActions,
 };
 
+// with node label control
+const SimpleWithControlData = genNetwork(
+  20,
+  true,
+  true,
+  DEFAULT_PLOT_SIZE,
+  DEFAULT_PLOT_SIZE
+);
+
+export const SimpleWithControl: Story<TemplateProps> = () => {
+  // list of node labels
+  const labels = SimpleWithControlData.nodes
+    .flatMap((node) => {
+      return node.label != null ? node.label : undefined;
+    })
+    .filter((data) => data != null);
+
+  // all values are selected as default
+  const [selectedNodeLabels, setSelectedNodeLabels] =
+    useState<(string | undefined)[]>(labels);
+  // show/hide node labels button
+  const [showNodeLabels, setShowNodeLabels] = useState<boolean>(false);
+
+  const ref = useRef<any>(null);
+
+  // custom switch based on MUI
+  const AntSwitch = withStyles((theme: Theme) =>
```

Comment:
If I remmeber correctly, i think coreui has a custom toggle/switch. Is there a reason not to use that one here?
(I do like the way this looks btw, just curious)

Reply:
👋 @asizemore Yes there exists a CoreUI's toggle/switch. But if I understand cocrrectly, it does not support labels at both sides (left and right of a toggle/switch UI), like **Hide** <toggle/switch UI> **Show**, which was suggested at the screenshot of  https://github.com/VEuPathDB/web-monorepo/issues/1108. That was the reason why I had to introduce a new switch based on Material-UI. Please correct me if I am incorrect.

Reply:
@asizemore with the new commits, MUI one is replaced with React-Select so MUI-related ones are removed.